CMakeLists.txt

```diff
@@ -98,11 +99,32 @@ set(LIBCOSIM_EXPORT_TARGET "${PROJECT_NAME}-targets")
 # ==============================================================================
 
 if(LIBCOSIM_USING_CONAN)
-    if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-        include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-        conan_basic_setup(NO_OUTPUT_DIRS)
+
+    if(NOT LIBCOSIM_USING_CONAN_AUTO_CONFIG OR CONAN_EXPORTED) # in conan local cache
```

Comment:
Please add some documentation that explains what happens when autoconfig is enabled. Either above this line (in which case it should describe the whole `if`/`else` block) or add a separate explanation for the `else` part below. Right now, only the default choice is explained, and that's the *least* confusing part, IMO.